rpc/src/module/net.rs

```diff
@@ -719,8 +725,21 @@ impl NetRpc for NetRpcImpl {
         let state = chain.shared().state();
         let (fast_time, normal_time, low_time) = state.read_inflight_blocks().division_point();
         let best_known = state.shared_best_header();
+        let min_chain_work = {
+            let mut min_chain_work_500k_u128: [u8; 16] = [0; 16];
+            min_chain_work_500k_u128.copy_from_slice(&MIN_CHAIN_WORK_500K.to_le_bytes()[..16]);
+            u128::from_le_bytes(min_chain_work_500k_u128)
+        };
```

Comment:
`MIN_CHAIN_WORK` is a value of type u256, however, ckb_jsonrpc does not support Uint256, so I need to cast it to u128 first and then convert it to Uint128.

Cc: @quake